### DISCUSSION TITLE
TPI 1 Macowins

### DISCUSSION THREAD

**Discussion by AlejoGol** [dds-jv/foro#342] [Consultas generales] [answered]
Hola, buenas, tengo algunas dudas sobre el ejercicio de macowins que hay que subir al classroom.
Primero, hay que subir el diagrama y los requerimientos aparte del código?
Segundo, nos podemos adaptar a la solución vista en clase (o usar alguna cosa de ella que tiene mas sentido que lo que habíamos pensado originalmente), o hay que apegarse a la solución original que habíamos pensado?
Muchas gracias

**Comment by gastonprieto** [comment, marked as answer]
Hola @AlejoGol 

No hace falta subir el diagrama y los requerimientos, esta es una parte de práctica de programación para bajar las ideas vistas en clase.

Pueden usar la solución que prefieran.

**Reply by gastonprieto** [reply]
esto te sirvio @AlejoGol ?
